{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "@allure.feature('test_broken')\n    def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\nTest_allure.py:30: Exception"}, "start": 1714032218579, "stop": 1714032218579, "uuid": "3e831550-ce24-4731-b72f-a81ddc97db9f", "historyId": "1a0939986f1b1a2e245e1bcaef4de2de", "fullName": "TestAllure.Test_allure#test_broken", "labels": [{"name": "feature", "value": "test_broken"}, {"name": "parentSuite", "value": "TestAllure"}, {"name": "suite", "value": "Test_allure"}, {"name": "subSuite"}, {"name": "host", "value": "JM200601001481"}, {"name": "thread", "value": "212384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestAllure.Test_allure"}]}